### PR TITLE
Add recover expansion failure docs

### PR DESCRIPTION
**Pull request by gnufied**
Add docs for volume expansion failure.

xref https://github.com/kubernetes/enhancements/issues/1790

### CODE REVIEW

**Comment by kbhawkey** [comment]
Hello @gnufied .
Would you rebase your changes? Thanks!

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -254,6 +254,19 @@ Expanding EBS volumes is a time-consuming operation. Also, there is a per-volume
...
 
 
+#### Recover from volume expansion failure
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
+
+
+Once user requests expansion of a PVC then Kuberneretes attempts to expand the underlying volume continuously until
+volume is expanded to desired size. In some cases if underlying storage system can not provision requested capacity
+then it may be desirable to retry expansion with a lower value.
```
Comment (tengqm) [inline content/en/docs/concepts/storage/persistent-volumes.md:264]: Need some copy editing from a native speaker I believe.

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -254,6 +254,19 @@ Expanding EBS volumes is a time-consuming operation. Also, there is a per-volume
...
+
+To support this if feature-gate `RecoverExpansionFailure` is enabled then a user may reduce requested capacity upto value in `pvc.Status.Capacity`
+and retry volume expansion with a lower value.
```
Comment (tengqm) [inline content/en/docs/concepts/storage/persistent-volumes.md:267]: This description is not clear.
How is a user supposed to request a reduction of the requested capacity? By merely reducing the requested capacity or by specifying a delta value? The "upto value" is confusing.

**Comment by annajung** [comment]
Enhancement is not being tracked for 1.19 
/milestone clear

**Comment by jimangel** [comment]
@gnufied given that this PR is not targeted for the `1.19` release, I will close it now. If the plan is to have this land in `1.20` you may reopen this PR again targeting the new dev-1.20 branch once created. Thanks!
/close

**Pull request closed without merging**